test/cypress/integration/ConfigPage.spec.js

```diff
@@ -0,0 +1,25 @@
+describe('Config Page', () => {
+	beforeEach(() => {
+		cy.visit('/config')
+		cy.injectAxe()
+	})
+
+	it('should load page', () => {
+		cy.get('h1').should('contain', 'Config Page')
```

Comment:
I'm guessing this is what required the update to `Nav.tsx` - this can be updated to `h4` since the `variant` is specified as `h4` in the Nav source

```suggestion
		cy.get('h4').should('contain', 'Config Page')
```